lib/timestamps.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function timestamps (data) {
+	const format = (year, month, day) => {
+		const monthName = new Date(year, month)
+			  .toLocaleDateString('en-GB', { month: 'short' })
+			  .toUpperCase();
+		return `${monthName} ${day}, ${year}`;
```

Comment:
why aren't you just returning `date.toLocaleString('en-US', { day: "numeric", month: "long", year: "numeric" });`?